web-ui/.prettierrc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "trailingComma": "none"
+  "trailingComma": "none",
+  "endOfLine" : "crlf"
```

Comment:
Current linefeed for all file is `lf` https://github.com/geonetwork/core-geonetwork/blob/main/.editorconfig#L4 so probably try with it on windows.